Conversion-Class Temperatura

```diff
@@ -0,0 +1,18 @@
+package com.company;
+
+public class Temperatura {
+
+    public double temperatura;
+    public void setTemperatura(double temperaturaZilei) {
+        this.temperatura = temperaturaZilei;
+    }
+    public double getTemperatura(){
+        return temperatura;
+    }
+
+    public void temperatura (double c, double f) {
+        double celsius = 0;
+        double fahrenheit = 70;
+        double temperatura = 5/9*(f-32);
```

Comment:
Where are you using the `c` parameter? If you want to override the temperatura attribute in this final line you can do it with the setter defined above `setTemperatura`